pkgs/applications/virtualization/docker/default.nix

```diff
@@ -39,7 +39,7 @@ stdenv.mkDerivation rec {
   buildPhase = ''
     patchShebangs .
     export AUTO_GOPATH=1
-    export DOCKER_GITCOMMIT="a34a1d59"
```

Comment:
Has this change any relationship with the PR? Maybe we should use something fixed like `deadbeef`